teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,28 @@
-try:
-    import pycodestyle as pep8
-except ImportError:
-    import pep8
+from flake8.formatting import base
 
 import re
 
 from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
```

Comment:
I see, it requires --format=teamcity-messages now. Is it possible to report messages without specifying a custom formatter?